docs/releasenotes/5.3.0.rst

```diff
@@ -39,6 +39,22 @@ and size, new method ``ImageOps.pad`` pads images to fill a requested aspect
 ratio and size, filling new space with a provided ``color`` and positioning the
 image within the new area through a ``centering`` argument.
 
+Image Size
+==========
+
+If you attempt to set the size of an image directly. e.g.
+``im.size = (100, 100)``, you will now receive an AttributeError. This is not
```

Comment:
Let's use code formatting for the error (and below).

Reply:
Okay, done